pyproject.toml

```diff
@@ -22,15 +23,18 @@ dependencies = [
 requires = ["maturin>=1.7,<2.0"]
 build-backend = "maturin"
 
-# target torch to cuda 12.8
+# target torch to rocm 6.3
 [tool.uv.sources]
 torch = [
-    { index = "pytorch-cu128" },
+    { index = "pytorch-rocm63" },
```

Comment:
ROCm 6.4 is the latest supported one, and there's also ROCm 7. I suggest switching to TheRock wheels for best compatibility https://github.com/ROCm/TheRock/blob/main/RELEASES.md#installing-pytorch-python-packages - these also install ROCm itself as pip wheels, so you don't need to worry about whether the system contains a specific version of ROCm.